### PR TITLE
 [TE] frontend - aaronucsd/added initial home dashboard page

### PR DESCRIPTION
**Pull request by aaronucsd**
screen shot 2018-04-05 at 12 59 52 pm
screen shot 2018-04-05 at 12 59 23 pm

1. Added new home page
2. Updated to correct the anomalies mappy to latest design (alerts inside table)
3. Updated custom templates for the dashboard table
4. Updatd stats-cards to not display NaN
5. UPdated fixture mock for application anomalies
6. TODO tests to follow since this is poc stage still

### CODE REVIEW

File: thirdeye/thirdeye-frontend/app/pods/manage/alerts/index/route.js [hunk 1]
```diff
@@ -68,10 +68,10 @@ export default Route.extend({
...
-    error(error, transition) {
-      if (error) {
-        return this.transitionTo('error');
-      }
-    }
+    // error(error, transition) {
+    //   if (error) {
+    //     return this.transitionTo('error');
+    //   }
+    // }
```
Comment (apucher) [inline thirdeye/thirdeye-frontend/app/pods/manage/alerts/index/route.js]: you may want to remove this route change

Comment (aaronucsd) [reply thirdeye/thirdeye-frontend/app/pods/manage/alerts/index/route.js]: Ok i can remove for now. I want to add it back once I update the error handling more.

Comment (aaronucsd) [reply thirdeye/thirdeye-frontend/app/pods/manage/alerts/index/route.js]: done.

File: thirdeye/thirdeye-frontend/app/styles/pods/home/alerts-dashboard.scss [hunk 2]
```diff
@@ -1,16 +1,35 @@
-.application-header {
-  display: flex;
-  justify-content: space-between;
-  align-items: baseline;
+.dashboard-container {
+  &__body {
+    background-color: #FFF;
+    margin-bottom: 24px;
+    padding: 0 15px;
+    border: 1px solid rgba(0, 0, 0, 0.1);
```
Comment (justYves) [inline thirdeye/thirdeye-frontend/app/styles/pods/home/alerts-dashboard.scss]: fyi @newsummit created an `app-shade` mixin 

https://github.com/linkedin/pinot/blob/master/thirdeye/thirdeye-frontend/app/styles/shared/_functions.scss#L2

File: thirdeye/thirdeye-frontend/app/styles/pods/home/alerts-dashboard.scss [hunk 3]
```diff
@@ -1,16 +1,35 @@
...
   }
-}
 
-.anomalies-table {
-  &__column-cell {
+  &__redirect-link{
     display: flex;
-    align-items: center;
+    justify-content: center;
+    border: solid 1px #CFCFCF;
```
Comment (justYves) [inline thirdeye/thirdeye-frontend/app/styles/pods/home/alerts-dashboard.scss]: Let's not hard code any colors. 

We should utilize those we have here: https://github.com/linkedin/pinot/blob/master/thirdeye/thirdeye-frontend/app/styles/wrapper/_color.scss

File: thirdeye/thirdeye-frontend/app/utils/anomaly.js [hunk 4]
```diff
@@ -47,13 +49,50 @@ export function updateAnomalyFeedback(anomalyId, feedbackType) {
  * Fetch a single anomaly record for verification
  * @method verifyAnomalyFeedback
  * @param {Number} anomalyId
- * @return {undefined}
+ * @return {Ember.RSVP.Promise}
  */
 export function verifyAnomalyFeedback(anomalyId) {
-  const anomalyUrl = getAnomalyDataUrl() + anomalyId;
-  return fetch(anomalyUrl).then(checkStatus);
+  const url = anomalyApiUrls.getAnomalyDataUrl() + anomalyId;
```
Comment (justYves) [inline thirdeye/thirdeye-frontend/app/utils/anomaly.js]: nit: let's use template literal here.

File: thirdeye/thirdeye-frontend/app/styles/pods/home/alerts-dashboard.scss [hunk 5]
```diff
@@ -1,16 +1,35 @@
-.application-header {
-  display: flex;
-  justify-content: space-between;
-  align-items: baseline;
+.dashboard-container {
+  &__body {
+    background-color: #FFF;
+    margin-bottom: 24px;
```
[the remark this answers is not in the document]
Comment (aaronucsd) [reply thirdeye/thirdeye-frontend/app/styles/pods/home/alerts-dashboard.scss]: Great. I was wondering where these were. Thanks.

File: thirdeye/thirdeye-frontend/app/styles/pods/home/alerts-dashboard.scss (hunk 2 above)
Comment (aaronucsd) [reply thirdeye/thirdeye-frontend/app/styles/pods/home/alerts-dashboard.scss]: Sure. Will use this.

File: thirdeye/thirdeye-frontend/app/pods/components/stats-cards/component.js [hunk 6]
```diff
@@ -57,7 +57,7 @@ export default Component.extend({
 
       card.forEach((stat, index) => {
         const property = props[index];
-        obj[property] = stat;
+        obj[property] = (property === 'value' && isNaN(stat)) ? 0 : stat;//let's return a number vs NaN
```
[the remark this answers is not in the document]
Comment (aaronucsd) [reply thirdeye/thirdeye-frontend/app/pods/components/stats-cards/component.js]: Done

File: thirdeye/thirdeye-frontend/app/pods/home/route.js [hunk 7]
```diff
@@ -27,65 +47,107 @@ export default Route.extend({
...
-      if (!anomalyMapping[metricName]) {
-        anomalyMapping[metricName] = {};
-      }
+  model(params) {
+    const { appName, startDate, endDate } = params;
+    return hash({
+      appName,
+      startDate,
+      endDate,
+      applications: fetch('/thirdeye/entity/APPLICATION').then(checkStatus)
```
[the remark this answers is not in the document]
Comment (aaronucsd) [reply thirdeye/thirdeye-frontend/app/pods/home/route.js]: Actually just realized there is already existing one in `app/utils/api/self-serve.js`

const allApplications = '/thirdeye/entity/APPLICATION';

File: thirdeye/thirdeye-frontend/app/styles/pods/home/alerts-dashboard.scss (hunk 3 above)
Comment (aaronucsd) [reply thirdeye/thirdeye-frontend/app/styles/pods/home/alerts-dashboard.scss]: DONE

**Pull request merged**